VectorRumble.Desktop/VectorRumble.Desktop.csproj

```diff
@@ -22,8 +23,9 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
-    <PlatformTarget>x64</PlatformTarget>
+    <PlatformTarget>x86</PlatformTarget>

```

Comment:
is this the x64 release config? If so we need to change this back.

Reply:
oops

Reply:
@dellis1972 that was in the `Debug|x86` conditional. So looks right to me. Not sure why the x86 conditional is targeting `x64`. The `x64` conditionals target the right things.

Reply:
ah ok. no problem.